### ISSUE TITLE
Feature: Resume Training

### ISSUE THREAD

**Issue by Poly2it** [ArthurBrussee/brush#29]
Unless I am missing something, it is currently not possible to pause a training job and resume it later. This would be quite beneficial for splitting large jobs on limited resources, and it would also provide a safeguard against crashes.

**Comment by ArthurBrussee** [comment]
Yes great point. It is somehwat-sorta-kinda possible if you export a ply, and put it in your training data as an "init.ply".

That of course needs some improvements like:
- Automatically save a splat every N steps
- Save settings (https://github.com/ArthurBrussee/brush/issues/25) and other training data
- Save optimizer state (though irrelevant before https://github.com/ArthurBrussee/brush/issues/30)
- Do this in a way so that it is just included in the training data, and you can easily restart from it. That is... very annoying before https://github.com/ArthurBrussee/brush/issues/1 is fixed.

Burn has a checkpointing mechanism so could integrate with that, but I do like the idea that exports are just viewable ply files (mostly).

**Comment by ArthurBrussee** [comment]
Closing this in favour of https://github.com/ArthurBrussee/brush/issues/176
